core/common/src/main/java/alluxio/Constants.java

```diff
@@ -52,7 +52,7 @@
   public static final String MESOS_RESOURCE_DISK = "disk";
   public static final String MESOS_RESOURCE_PORTS = "ports";
 
-  public static final int SECOND_NANO = 1000000000;
+  public static final long SECOND_NANO = 1000000000;
```

Comment:
It can easily cause bugs by writing this code:

long time = System.nanoTime();
time = time + 10 * Constants.SECOND_NANO (overflow, it should be time + 10L * Constants.SECOND_NANO)